test/martha/martha_v3.test.js

```diff
@@ -458,8 +460,6 @@ test.serial('martha_v3 should return 500 if Data Object resolution fails', async
     t.is(response.statusCode, 500);
     t.is(response.body.status, 500);
     t.is(response.body.response.text, 'Received error while resolving DRS URL. Data Object Resolution forced to fail by testing stub');
-
-    sinon.assert.callCount(getJsonFromApiStub, 1);
```

Comment:
For this case (and the 2 below), I'm proposing that we don't really care how many times the stub was called as long as the error that the stub throws is represented in the response. Please speak up if you disagree. This was fallout from changing the order of fetching the SA key.

Reply:
IMHO this seems fine for these unhappy path cases since the error message is checked specifically